### PR TITLE
Update CONTRIBUTORS.txt

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ needs to be listed here.
 - Swift Server Work Group
 ### Contributors
 - Adam Fowler <adamfowler71@gmail.com>
+- Hassan Talat <htalat@users.noreply.github.com>
 - Kenta Kubo <601636+kkk669@users.noreply.github.com>
```
[the remark this answers is not in the document]
Comment (kkk669) [reply CONTRIBUTORS.txt:11]: Please use this.

kabuto669 at icloud.com

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@ needs to be listed here.
 - Swift Server Work Group
 ### Contributors
 - Adam Fowler <adamfowler71@gmail.com>
+- Hassan Talat <htalat@users.noreply.github.com>
```
[the remark this answers is not in the document]
Comment (htalat) [reply CONTRIBUTORS.txt:10]: @adam-fowler I would prefer: hassan.talat.92 @gmail.com

**Comment by adam-fowler** [comment]
@0xTim seems I expected everyone to know you by your first name only (probably true though)

**Comment by 0xTim** [comment]
@adam-fowler yeah I think my Git author settings just have Tim but happy to have it as my full name!

File: CONTRIBUTORS.txt [hunk 3]
```diff
@@ -7,11 +7,13 @@ needs to be listed here.
 - Swift Server Work Group
 ### Contributors
 - Adam Fowler <adamfowler71@gmail.com>
-- Kenta Kubo <601636+kkk669@users.noreply.github.com>
+- Hassan Talat <hassan.talat.92@gmail.com>
+- Kenta Kubo <kabuto669@icloud.com>
 - Kim de Vos <kimdevos12@hotmail.com>
 - Saleem Abdulrasool <compnerd@compnerd.org>
 - Steven Van Impe <steven.vanimpe@icloud.com>
 - Tim <0xtimc@gmail.com>
```
Comment (0xTim) [inline CONTRIBUTORS.txt:15]: I seem to be duplicated here

Comment (adam-fowler) [reply CONTRIBUTORS.txt]: sorted

**Review by 0xTim — APPROVED** [review]

**Comment by htalat** [comment]
I’d prefer hassan.talat.92 @ gmail.com

**Pull request merged**